examples/inverse/plot_custom_inverse_solver.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+"""
+================================================
+Source localization with a custom inverse solver
+================================================
+
+The objective of this example is to show how to plug a custom inverse solver
+in MNE in order to facilate empirical comparison with the methods MNE already
+implements (wMNE, dSPM, sLORETA, LCMV, (TF-)MxNE) etc.
+
+This script is educational and shall be used for methods
+evaluations and new developments. It is not meant to be an example
+of good practice to analyse your data.
+
+The example makes use of 2 functions ``apply_solver`` and ``solver``
+so changes can be limited to the ``solver`` function in order to
```

Comment:
Something more like "... to the ``solver`` function (which only takes three parameters: the whitened data, the gain matrix, and the number of orientations)" would help clarify why it's nice